src/main/java/core/basesyntax/service/impl/DataWriterImpl.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.DataWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+
+public class DataWriterImpl implements DataWriter {
+
+    @Override
+    public boolean writeDataToFile(List<String> list, String filename) {
+        try {
+            Files.write(Path.of(filename), list);
+            return true;
+        } catch (IOException e) {
+            throw new RuntimeException("Can`t write data to file");
```

Comment:
```suggestion
            throw new RuntimeException("Can`t write data to file " + fileName);
```
to make a message more informative I would recommend to add some kind of identifier of where the problem occurred